RockWeb/App_Code/GetFile.ashx.cs

```diff
@@ -39,84 +38,21 @@ public class GetFile : IHttpAsyncHandler
         /// <param name="extraData">Any state data needed to process the request.</param>
         /// <returns>An IAsyncResult that contains information about the status of the process.</returns>
         public IAsyncResult BeginProcessRequest( HttpContext context, AsyncCallback cb, object extraData )
-        {
-            try
-            {
-                // Check to see if this is a BinaryFileType/BinaryFile or just a plain content file (if isBinaryFile not specified, assume it is a BinaryFile)
-                bool isBinaryFile = ( context.Request.QueryString["isBinaryFile"] ?? "T" ).AsBoolean();
```

Comment:
I did find one spot in core that was using IsBinaryFile.  It's the CMS > FileManager, if a person were to click on the link on the FileUploader after uploading a file.  However, it is broken :).  Like you mentioned, it could just use a regular link.  FileUploader.cs would need to be fix though.  We could probably just fix FileUploader.cs LIne 648 to be `_aFileName.HRef = ResolveUrl( this.UploadedContentFilePath );`